web/src/test/java/com/microsoft/applicationinsights/web/internal/WebRequestTrackingFilterTests.java

```diff
@@ -74,7 +64,7 @@ public void testFilterInitializedSuccessfullyFromConfiguration() throws ServletE
         Filter filter = createInitializedFilter();
         WebModulesContainer container = ServletUtils.getWebModuleContainer(filter);
 
-        Assert.assertNotNull("Container shouldn't be null", container);
+        assertNotNull("Container shouldn't be null", container);
         Assert.assertTrue("Modules container shouldn't be empty", container.getModulesCount() > 0);
```

Comment:
I would like consistency if possible, can we stay consistent in importing asserts. Here Assert.assertFalse is for example imported and Assert.assertTrue isn't.

Reply:
We can clean up consistency issue like this in a style related PR.